hm_pyhelper/hardware_definitions.py

```diff
@@ -299,6 +327,8 @@ def is_raspberry_pi() -> bool:
     # Nebra Hotspot HAT Tinkerboard 2
     'NEBHNT-HHTK2': {
         'FRIENDLY': 'Nebra Hotspot HAT Tinkerboard 2',
+        'CPU_ARCH': 'armv7hf',
```

Comment:
I don't think Balena currently supports the Asus tinkerboard 2

Reply:
will remove it in that case.

Reply:
all the CamelCase and capital case variant names are candidates for removal. They are either alias to maintain compatibility or we don't care to support anymore.